builtin/game/chat.lua

```diff
@@ -329,13 +329,13 @@ core.register_chatcommand("setpassword", {
 				" password of " .. toname .. ".")
 
 		return true, "Password of player \"" .. toname .. "\" " .. act_str_past
-	end,
+	end
```

Comment:
While the comma is indeed not needed here, removing it does not improve anything.